packages/flutter/lib/src/painting/edge_insets.dart

```diff
@@ -490,11 +493,66 @@ class EdgeInsets extends EdgeInsetsGeometry {
   /// See also:
   ///
   ///  * [deflateSize], to deflate a [Size] rather than a [Rect].
```

Comment:
@HansMuller  i encourage you to feel empowered to get people to embrace the yak shave when you see something that isn't ideal, rather than adding more unideal stuff. :-)